src/twilio/tests/test.bal

```diff
@@ -73,6 +73,27 @@ function testSendSms() {
     }
 }
 
+@test:Config {
+    groups: ["basic"],
+    dependsOn: ["testAccountDetails"]
+}
+function testSendWhatsAppMessage() {
+    io:println("\n ---------------------------------------------------------------------------");
+    log:printInfo("twilioClient -> sendWhatsAppMessage()");
+
+    string fromMobile = config:getAsString("SAMPLE_WHATSAPP_SANDBOX");
+    string toMobile = config:getAsString("SAMPLE_TO_MOBILE");
+    string message = config:getAsString("SAMPLE_MESSAGE");
```

Comment:
Please add these configurations to the [tests README file](https://github.com/wso2-ballerina/module-twilio/blob/master/src/twilio/tests/Readme.md)

Reply:
Added the newly added configuration parameters to [tests README file](https://github.com/wso2-ballerina/module-twilio/blob/master/src/twilio/tests/Readme.md).